### ISSUE TITLE
Plugin ADB Idea depends on unknown plugin org.jetbrains.android

### ISSUE THREAD

**Issue by Succelos** [pbreault/adb-idea#97]
Hello,

I'm trying to install this plugin on Rider 2018.2.3 with "Install plugin from disk..." but I get a warning saying "Plugin ADB Idea depends on unknown plugin org.jetbrains.android".

![capture du 2018-10-18 18-04-38](https://user-images.githubusercontent.com/32650971/47168105-54be0480-d300-11e8-9b96-3cfaab46667a.png)

Unfortunately, I can't find this plugin anywhere.

Did I miss something in my settings ?

**Comment by JMartelot** [comment]
Hi,

I'm facing the same issue, I'm working on Debian 9.

Could you advise us, how to fix it?

Thanks

**Comment by pbreault** [comment]
It probably can't work. This plugin depends on the Android tooling that is based on Gradle. This is where it gets access to the debug bridge and to the app's package name.

I'm assuming that you are using Xamarin, which is probably completely independent from the android tooling present in Intellij and Android Studio.

Did the plugin appear in the plugin repository or did you manually install it?

**Comment by Succelos** [comment]
Yes, the plugin appeared in the plugin repository but it didn't work either, that's why I tried to install it manually.

Indeed, I am using Xamarin, I had some issues with it, so I moved all the build process to a script using a docker image which generate an APK file. Using adb tool, I can make it run but I would like to be able to debug it from Rider. Do you maybe know a work-around to do so ?

**Comment by Succelos** [comment]
Yes, the plugin appeared in the plugin repository but it didn't work either, that's why I tried to install it manually.

Indeed, I am using Xamarin, I had some issues with it, so I moved all the build process to a script using a docker image which generate an APK file. Using adb tool, I can make it run but I would like to be able to debug it from Rider. Do you maybe know a work-around to do so ?
